ogre2/include/ignition/rendering/ogre2/Ogre2RenderTarget.hh

```diff
@@ -133,7 +141,10 @@ namespace ignition
           Ogre::CompositorWorkspace *_workspace,
           const std::string &_workspaceDefName,
           const std::string &_baseNode, const std::string &_finalNode,
-          const std::vector<RenderPassPtr> &_renderPasses, bool _recreateNodes);
+          const std::vector<RenderPassPtr> &_renderPasses,
+          bool _recreateNodes,
+          Ogre::Texture *(*_ogreTextures)[2],
+          bool _isRenderWindow);
```

Comment:
add doxygen comments for these new params

we'll need to keep the existing function since adding the args here breaks API/ABI.